t/geturl_connectivity.pl.in

```diff
@@ -13,6 +13,9 @@
     );
     defined($ipv6_socket);
 };
+my $has_curl = eval {
+    qx{ @CURL@ --version 2>/dev/null; } && $? == 0;
+};
```

Comment:
Why is this wrapped in an `eval`?

Reply:
Looks like not necessary.  I have removed it.